src/main/java/core/basesyntax/Drawable.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Drawable {
+    String draw();
```

Comment:
`To 'draw' means to print out all information about a figure using System.out.println() (you shouldn't override toString() method for this).` check task 
```suggestion
    void draw();
```